### PR TITLE
Test districts

### PR DESCRIPTION
**Pull request by nbdavies**
The rest of the pieces needed to start testing district in the feature tests, to fix https://github.com/openelections/openelections-data-wi/issues/42.

To explain the changes in steps.py:

Values from a line in the feature file get interpolated into a sentence:
`I search for <party> party candidate <candidate> running for <office> dist. <district> in the <ward> in <county>`
Then `behave` extracts the values from that sentence by comparing it to the `@when` string. It seems to effectively use the filler words of the sentence as delimiters.

When party column is missing from a particular section, it's left un-substituted in the sentence as `"<party>"`. When the district column is present for the table as a whole, but left blank for a particular line, then `behave` doesn't recognize that the sentence even matches the `@when` clause at all, and throws a NotImplementedError. 

There's still a failing test due to #45.

### CODE REVIEW

File: tests/features/wi-elections.feature [hunk 1]
```diff
@@ -89,10 +89,6 @@ Feature: WI Elections
     | candidate                             | county    | office                                    | district  | ward                                          | votes | total |
     | Ken Sortedahl                         | St. Croix | St Croix County Circuit Court, Branch 4   |           | VILLAGE OF DEER PARK                          | 3     | 50    |
 
-  Examples: 20080219__wi__primary__ward.csv
```
Comment (davipo) [inline tests/features/wi-elections.feature]: Why is this test deleted?

Comment (nbdavies) [reply tests/features/wi-elections.feature]: Not sure what happened there. I've added it back in

**Comment by davipo** [comment]
I began merging this PR and found problems. I corrected them in my branch dp-add-district-to-tests, and pushed it here to GitHub. Then I resolved other merge conflicts (editing the feature tests) here on GitHub. Now GitHub displays a commit "Merge branch 'dp-add-district-to-tests' into test-districts". I'm confused, I thought this PR is going the other way. Maybe this commit is merging in the commits I pushed, but they are not shown.

The tests did not pass because the `wi-elections_2018_sample_with_district.feature` file was restored in a merge after having been deleted earlier. This file had dummy test data, and the testing framework runs every file with a `.feature` extension.

This PR now shows no conflicts, but still needs merging. Trying that again.

**Comment by davipo** [comment]
Merged, then removed dummy test data: tests pass.

**Pull request merged**